### ISSUE TITLE
Docker Errors on Load

### ISSUE THREAD

**Issue by jamesward** [sbt/sbt-native-packager#1065]
## Expected behaviour

When sbt loads a project containing the sbt-native-packager plugin, it should not produce docker errors.

## Actual behaviour

On project load I get docker errors:
```
$ sbt
[info] Loading settings from plugins.sbt ...
[info] Loading project definition from /Volumes/Home/j.ward/projects/get-you-a-license/project
[info] Loading settings from build.sbt ...
Get http:///var/run/docker.sock/v1.20/version: dial unix /var/run/docker.sock: no such file or directory.
* Are you trying to connect to a TLS-enabled daemon without TLS?
* Is your docker daemon up and running?
[info] Set current project to get-you-a-license (in build file:/Volumes/Home/j.ward/projects/get-you-a-license/)
[info] sbt server started at 127.0.0.1:4206
[get-you-a-license] $
```

## Information

- What sbt-native-packager are you using: 1.3.1 via Play 2.6.7
- What sbt version: 1.0.3
- What is your build system (e.g. Ubuntu, MacOS, Windows, Debian ): MacOS
- What package are you building (e.g. docker, rpm, ...): Universal

**Comment by dwickern** [comment]
This was fixed in #1058, please try with sbt-native-packager 1.3.2

**Comment by jamesward** [comment]
Thanks! Sorry I didn't see that one.  Guess I searched for the wrong terms.
